script.js

```diff
@@ -48,9 +50,9 @@ function displayBooks(books) {
         <small> <strong>Published</strong>: ${publishdate}</small>
       </div>
       <a href = "${previewLink}" class ="book-link" target="_blank">  Google Books Preview </a>
-       `;
+       `);
```

Comment:
Because jQuery's syntax for creating new nodes is gross, I just set all the HTML contents of the newly-created `<div>` all at once. This can have security implications, so it's not good for all situations.

If you find yourself needed to create DOM nodes, and using React (or similar) would be overkill, a template library like [Handlebars](https://handlebarsjs.com/) could be very useful.